v2/samples/apimanagement/v1api20230501preview/v1api20230501preview_authorizationprovidersauthorization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apimanagement.azure.com/v1api20230501preview
+kind: AuthorizationProvidersAuthorization
+metadata:
+  name: sampleapimauthorization
+  namespace: default
+spec:
+  authorizationType: "OAuth2"
+  error:
+    code: "samplecode"
+    message: "samplemessage"
+  oauth2grantType: "AuthorizationCode"
+  status: "samplestatus"
```

Comment:
Same q about status and error here -- are they required for the sample? If no, IMO remove them

Reply:
https://learn.microsoft.com/en-us/rest/api/apimanagement/authorization/create-or-update?view=rest-apimanagement-2022-08-01&tabs=HTTP, based on this, I think we should remove them as well